index.html

```diff
@@ -13,7 +13,7 @@
 
 <!-- Feel free to change this text here -->
 <p>
-  Fork me? Fork you, @octocat!
+  Fork me? Fork you, @rtryoda!
 </p>
 
 </body>
```

Comment:
*HQ,YYYYYYYYYY,V8,en,HHMMSS,S,latitude,D,longitude,G,speed,direction,DDMMYY,vehicle_status#
Note:VV=V8: send command to platform to get the location description in english, 2 bytes.
     en: get the location description in english.
     The other is same with above.
For Example send message to platform to get location description in english:
*HQ,1400046168,V8,en,075738,A,2234.3066,N,11351.6829,E,000.0,000,080813,FFFFFBFF#